api/internal/handler/ssl/ssl_test.go

```diff
@@ -0,0 +1,101 @@
+	  "cert": "-----BEGIN CERTIFICATE-----\nMIIEVzCCAr+gAwIBAgIQITiNM7xmudhg3pK85KDwLDANBgkqhkiG9w0BAQsFADB/\nMR4wHAYDVQQKExVta2NlcnQgZGV2ZWxvcG1lbnQgQ0ExKjAoBgNVBAsMIWp1bnh1\nY2hlbkBqdW54dWRlQWlyIChqdW54dSBjaGVuKTExMC8GA1UEAwwobWtjZXJ0IGp1\nbnh1Y2hlbkBqdW54dWRlQWlyIChqdW54dSBjaGVuKTAeFw0xOTA2MDEwMDAwMDBa\nFw0zMDA3MDgwNzQ4MDJaMFUxJzAlBgNVBAoTHm1rY2VydCBkZXZlbG9wbWVudCBj\nZXJ0aWZpY2F0ZTEqMCgGA1UECwwhanVueHVjaGVuQGp1bnh1ZGVBaXIgKGp1bnh1\nIGNoZW4pMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAxjtCfcaznJr4\nIaZKhyGBwsN+RU3yE+blzks3RePTNmQCgUMBRtKL2E/G3NIJTSwfoaQ4STy+d9ak\nj5dWTntFtaHz9btE+Wc3PUbm/5J+0TD6tAbEguid6xfq0BMIdTzPKlLg/A8We42j\ni4jDS+ksrmSD7EtZDzwbbIqLpHaOuurW4zTIsYUGyPVqYOeWm3V0RYd4u2bv/bIh\ndYwhnLLJIzmlDmOLGEN4uVlPxyCEYQPNl7iC/ksXd9j/GyotlE9bKhV0YwFUeykV\nqCFb5Jki6GW/bnFejLHHA60WAX+qy2yNbRY/Zdq2OhkxZU/us+yv2SOtAxov4bAX\n5Q3hEOVrsQIDAQABo3kwdzAOBgNVHQ8BAf8EBAMCBaAwEwYDVR0lBAwwCgYIKwYB\nBQUHAwEwDAYDVR0TAQH/BAIwADAfBgNVHSMEGDAWgBRU+EbJj+Hp62gCrNvb3yQk\nYnPHXDAhBgNVHREEGjAYgglyb3V0ZS5jb22CCyoucm91dGUuY29tMA0GCSqGSIb3\nDQEBCwUAA4IBgQAvKN2GEorAlx5sfUU2uiL49iFmQSMDLZQminQl1RIHTI/h+jz8\nNluZSdxDFmNq8am6B2ofD3VLl6StC/G+G6YuekPz+QrUNK4UB+8ftRmY4YRFGTQ6\nRnFli1wOq2ES9vPjKlIj77cznr8uwVHPHq8JxGbn/rx3oVDVPndXFCkJJ1DDjRT+\n22atHNzHt5bc9ut8Fq5NW61P+nnMMFShKJaPBkmm9Pf2pEOd8Y7OU8Iy1Kj65fsE\nUshGF5+RWoxdv6/9f6/uOQhmq3MEKqneUC3pjVZ8TiBlRvADxxR5krvujQswms0D\nFGpRMtGpPGMWTuptSIMwNcar/luVig7wGIBeV5ZaOlSOx3911le9mlS7+2lLqf5H\n5dsMkP30Sjv/jfrIL+SE1qeK3kjL0iIwA/PPARvhctExs9y2llT9+drbJofZUi+I\nZdYfAfyJT4htbcl7jHN8oY7vzwgTyxCcBxkbqKfBqabneutj0jfX39zP0G696tiZ\ndQFXCS4wkvw0CG0=\n-----END CERTIFICATE-----"
+  }`
+	json.Unmarshal([]byte(reqBody), ssl2)
+	ctx.SetInput(ssl2)
+	_, err = handler.Update(ctx)
+	assert.Nil(t, err)
+
+	//list
+	listInput := &ListInput{}
+	reqBody = `{"page_size": 1, "page_number": 1}`
```

Comment:
Please use `page` instead of `page_number`, cc @ShiningRush @nic-chen

Reply:
Is there any special reason?
`page_number` is the more commonly used than `page`

Reply:
@ShiningRush  
Seems to modify here, please have a look:
https://github.com/ShiningRush/droplet/blob/master/data/page.go#L21

Reply:
> Is there any special reason?
> `page_number` is the more commonly used than `page`

1. it's strange to see URL like `https://xxxx.com/list?page_number=1&page_size=10`.
![image](https://user-images.githubusercontent.com/2106987/95812712-95628400-0d48-11eb-9060-a6ccdcac99bd.png)

2. this field will be passed to the frontend's pagination components, if using `page_number`, we have to transform it to  `page`, this transformer is not needed.

Reply:
Well, if we want to modify this name just for our own legacy reasons, we can add a dedicated `pager` to the `manager api` to meet individual needs.
The default pager of `droplet` is for common used.
 @nic-chen you can create a new struct to instead of it.

Reply:
It's not just for legacy reasons, `page` is commonly used in the frontend. BTW, snake_naming is not allowed in ESLint's rules by default actually, but I modify that rule to allow snake_naming, just for being compatible with APIs.

Reply:
done.

Reply:
Got it.
However, snake_case will be more general.
Can refer to [here](https://www.javacodegeeks.com/2018/12/whats-case-api.html)
About the `page_number`, emmm, it indeed hardly to say this is a common name, I have viewed some front-end  lib, the name about pagination is different.
Maybe I should remove concrete data struct from `droplet`, just keep interface.

Reply:
> `page` is commonly used in the frontend

agree +1